src/libs/API/types.ts

```diff
@@ -388,6 +389,7 @@ type WriteCommandParameters = {
     [WRITE_COMMANDS.UPDATE_POLICY_DISTANCE_RATE_VALUE]: Parameters.UpdatePolicyDistanceRateValueParams;
     [WRITE_COMMANDS.SET_POLICY_DISTANCE_RATES_ENABLED]: Parameters.SetPolicyDistanceRatesEnabledParams;
     [WRITE_COMMANDS.DELETE_POLICY_DISTANCE_RATES]: Parameters.DeletePolicyDistanceRatesParams;
+    [WRITE_COMMANDS.UPDATE_POLICY_CONNECTION_CONFIGURATION]: Parameters.UpdatePolicyConnectionConfigurationParams<never, never>;
```

Comment:
A way around the type error is this:

```suggestion
    [WRITE_COMMANDS.UPDATE_POLICY_CONNECTION_CONFIGURATION]: Parameters.UpdatePolicyConnectionConfigurationParams<any, any>;
```

I'm not sure if there is a better way, maybe there is 🤷 

This error I mean:
<img width="1097" alt="image" src="https://github.com/Expensify/App/assets/87341702/ad082dcb-8933-42cb-86ea-531484619ab8">